editor/src/clj/editor/workspace.clj

```diff
@@ -42,8 +43,17 @@ ordinary paths."
   (^File [workspace evaluation-context]
    (io/as-file (g/node-value workspace :root evaluation-context))))
 
+(defn- skip-first-char [^String path]
+  (.substring path 1 (count path)))
+
 (defn build-path [workspace]
-  (io/file (project-path workspace) "build/default/"))
+  (io/file (project-path workspace) (skip-first-char build-dir)))
+
+(defn plugin-path
+  (^File [workspace]
+   (io/file (project-path workspace) (skip-first-char plugins-dir)))
+  (^File [workspace path]
+   (io/file (project-path workspace) (str (skip-first-char plugins-dir) (skip-first-char path)))))
```

Comment:
Indent function bodies (`(io/file ...)` lines) with two spaces. There's probably a setting for this in Calva somewhere. It probably behaves differently for this because it has multiple arity overloads.

Reply:
Hmm, it doesn't seem to have any settings for that.
It does mention support for a`cljfmt` file. Is that a standard we should use?

Reply:
Yeah, that might do the trick. Really out-of-scope for this PR, but I do believe we set up a preflight script that uses `cljfmt`. Maybe this bit is the `cljfmt` config and just putting that map in a file would work? https://github.com/defold/defold/blob/dev/editor/tasks/leiningen/preflight/fmt.clj#L80-L89

Reply:
I'll look into the `cljfmt` now in a separate PR.